packages/snap/src/index.ts

```diff
@@ -0,0 +1,463 @@
+import { OnRpcRequestHandler } from "@metamask/snaps-types";
+import { panel, text, heading, divider, copyable } from "@metamask/snaps-ui";
+import { initializeChains } from "./initialize";
+import { Chain, Chains, Fees } from "./types/chains";
+import { Address } from "./types/address";
+import { ChainState, AddressState } from "./state";
+import { Result } from "./types/result";
+import { submitTransaction } from "./transaction";
+import { getAddress } from "./address";
+
+/**
+ * Handle incoming JSON-RPC requests, sent through `wallet_invokeSnap`.
+ *
+ * @param args - The request handler args as object.
+ * @param args.request - A validated JSON-RPC request object.
```

Comment:
I'm not sure if the request is already validated or not.  Maybe:

```suggestion
 * @param args.request - A JSON-RPC request object that will be validated.
```